datadog/tests/resource_datadog_integration_slack_channel_test.go

```diff
@@ -83,7 +83,7 @@ func testAccCheckDatadogIntegrationSlackChannelConfig_Create(uniq string) string
    `, uniq)
 }
 
-func testAccCheckDatadogIntegrationSlackChannelConfig_Update(uniq string) string {
+func testAccCheckDatadogIntegrationSlackChannelConfigUpdate(uniq string) string {
```

Comment:
Do cassettes need recording if these change?

Reply:
No it's private helper.

Reply:
Ahh right, I saw testAcc and assumed 🙂 
LGTM then 👍